src/pages/Component/setting/auto-deploy.js

```diff
@@ -213,8 +213,11 @@ export default class AutoDeploy extends PureComponent {
       status,
       tabLoading,
       support_type,
+      url,
       service_source
     } = this.state;
+    const setUrl = url.replace('http://127.0.0.1:5000', window.location.host);
```

Comment:
window.location.protocol
window.location.host
window.location.port

Reply:
fix the `${window.location.protocol}//${window.location.host}`